src/huggingface_hub/commands/lfs.py

```diff
@@ -162,33 +171,88 @@ def run(self):
                 }
             )
 
+            # caculate workload per thread
+            n_url = len(presigned_urls)
+            n_thread = min(n_url, ref_concurrency)
```

Comment:
I would just set a hard value here, either 64 or 128

```suggestion
            n_thread = min(n_url, 64)
```